ui/src/app/machines/views/MachineDetails/MachineTests/MachineTests.tsx

```diff
@@ -95,13 +108,17 @@ const MachineTests = (): JSX.Element => {
               ]) => {
                 const { model, name, serial } =
                   scriptResults[0]?.parameters?.storage?.value || {};
+                let title = name ? `/dev/${name}` : null;
+                if (name && model && serial) {
+                  title = `${title} (model: ${model}, serial: ${serial})`;
+                }
                 return (
                   <div key={physical_blockdevice}>
-                    <h5 data-test="storage-heading">
-                      {model && name && serial
-                        ? `/dev/${name} (model: ${model}, serial: ${serial})`
-                        : physical_blockdevice}
-                    </h5>
+                    {title && (
```

Comment:
Do we need this short circuit if we already have the fallback `physical_blockdevice` below?

Reply:
Ah yes, the `title || physical_blockdevice` was impossible to reach, I've removed that, thanks.